tests/foreman/destructive/test_leapp_satellite.py

```diff
@@ -49,3 +49,6 @@ def test_positive_leapp(target_sat):
     result = target_sat.execute('cat /etc/redhat-release | grep -Po "\\d"')
     # Check if RHEL was upgraded
     assert result.stdout[0] == str(orig_rhel_ver + 1), 'RHEL was not upgraded'
+    # Check satellite's health
+    sat_health = target_sat.execute('satellite-maintain health check')
```

Comment:
we can use f-m CLI for health check here
```suggestion
    sat_health = target_sat.cli.Health.check()
```

Reply:
Ladislav would not be here for some time, so let me commit your suggestion.

Reply:
@ogajduse Thanks, Could you open a cherrypick for 6.12.z and 6.13.z branches too as he isn't around?

Reply:
cherry-picked to 6.12 and 6.13:
#10438
#10439